### PR TITLE
LRCI-4234 generate relevant properties

### PR DESCRIPTION
**Pull request by CharlotteFrancis**

### CODE REVIEW

**Comment by kenjiheigel** [comment]
@CharlotteFrancis , can we ignore test.properties that come from `modules/test/jenkins-results-parser/src/test/resources/dependencies`? Those are for our unit tests and don't need any changes.

File: modules/apps/blogs/test.properties [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
 
-    modules.includes.required.test.batch.class.names.includes[relevant]=\
+    modified.files.includes[relevant][blogs-rule]=**
+
+    relevant.rule.names=\
+        blogs-rule
+
+    test.batch.names[relevant][blogs-rule]=\
+        functional-tomcat90-mysql57-jdk8,\
+        relevant=
```
Comment (kenjiheigel) [inline modules/apps/blogs/test.properties:12]: Not sure where this came from, but shouldn't be generated in.

File: modules/apps/dynamic-data-mapping/test.properties [hunk 2]
```diff
@@ -1,38 +1,44 @@
...
-    modules.includes.required.functional[relevant]=\
+    modified.files.includes[relevant][dynamic-data-mapping-rule]=**
+
+    relevant.rule.names=\
+        dynamic-data-mapping-rule
+
+    test.batch.names[relevant][dynamic-data-mapping-rule]=\
+        functional-tomcat90-mysql57-jdk8,\
+        functional-upgrade-tomcat90-mysql57-jdk8,\
+        relevant=
```
Comment (kenjiheigel) [inline modules/apps/dynamic-data-mapping/test.properties:13]: Another one, if that helps to triangulate where the issue is coming from.

File: modules/apps/portal-vulcan/test.properties [hunk 3]
```diff
@@ -1,19 +1,26 @@
...
-## Test Batch
+## Test Suites
 ##
 
+    modified.files.includes[relevant][portal-vulcan-rule]=**
+
+    relevant.rule.names=\
+        portal-vulcan-rule
+
+    test.batch.names[relevant][portal-vulcan-rule]
```
Comment (kenjiheigel) [inline modules/apps/portal-vulcan/test.properties:10]: Missing batch. Should be `functional-tomcat90-mysql57-jdk8`

File: modules/apps/portal-workflow/test.properties [hunk 4]
```diff
@@ -1,34 +1,47 @@
...
-## Test Batch
-##
+    modules.includes.required.test.batch.class.names.includes[modules-integration-mysql57-jdk8][relevant][portal-workflow-rule]=\
+        dxp/apps/portal-workflow/**/*Test.java
+
+    test.batch.names[relevant][portal-workflow-rule]=\
+        functional-tomcat90-mysql57-jdk8,\
+        functional-tomcat90-mysql57-jdk8,\
+        functional-upgrade-tomcat90-mysql57-jdk8,\
+        relevant=
```
Comment (brittneyq) [inline modules/apps/portal-workflow/test.properties:20]: relevant= is still here

File: modules/apps/questions/test.properties [hunk 5]
```diff
@@ -1,27 +1,41 @@
...
         apps/questions/**/*Test.java
 
-##
-## Test Batch
-##
+    modules.includes.required.test.batch.class.names.includes[modules-integration-mysql57-jdk8][relevant][questions-rule]=\
+        apps/message-boards/**/*Test.java,\
+        apps/questions/**/*Test.java
+
+    test.batch.names[relevant][questions-rule]=\
```
Comment (brittneyq) [inline modules/apps/questions/test.properties:18]: batch names are duplicated and there is still relevant=

**Pull request closed without merging**